test/OpenTelemetry.Tests/Metrics/InMemoryExporterTests.cs

```diff
@@ -23,8 +23,8 @@ namespace OpenTelemetry.Metrics.Tests
 {
     public class InMemoryExporterTests
     {
-        [Fact(Skip = "To be run after https://github.com/open-telemetry/opentelemetry-dotnet/issues/2361 is fixed")]
-        public void InMemoryExporterShouldDeepCopyMetricPoints()
+        [Fact]
```

Comment:
I don't think we need these tests. We have dedicated tests for each instrument already which use `InMemoryExporter` anyway.

Reply:
Do you think we should remove this entire test class?

In the tests I've added, I'm focused on the InMemoryExporter has asserting what was exported and I have notes describing what is expected and why.
These maybe useful as a demonstration of the InMemoryExporter's behavior. 
But I understand the overlap with the MetricAPITests.

Reply:
I think we can remove the entire test class. The other things such as testing the reference equality of `Metric` is indirectly tested anyway since we only use on instrument to emit metrics here.